src/firefly/js/api/ApiUtil.js

```diff
@@ -42,15 +49,11 @@ export function debug(...msg) {
 
 
 
-
-
-
 /**
  *
  * @param {string|Object} div a div element or a string id of the div element
  * @param {Object} Component a react component
  * @param {Object} [props] props for the react component
- * @namespace firefly
  * @Function renderDOM
```

Comment:
I hope these @function declaration are not necessary in these cases where it's clearly a function.